lib/rules/base.js

```diff
@@ -306,7 +308,7 @@ module.exports = class Base {
       }
 
       if (ruleHandler && !plugin.isDisabled()) {
-        ruleHandler.apply(plugin, arguments); // eslint-disable-line prefer-rest-params
+        maybeRuleHandlerResult = ruleHandler.apply(plugin, arguments); // eslint-disable-line prefer-rest-params
```

Comment:
this logic required to allow node replacement in fix mode example: (mustache -> element)